CHANGELOG.md

```diff
@@ -5,6 +5,18 @@
 * Breaking Changes
   * All SDK error enums are now internal
   * See [list of new / updated error cases and codes](SDK_ERROR_CODES.md)
+* BraintreeVenmo
+  * Allow merchants to collect enriched customer data if enabled in the Braintree Control Panel
+  * Add the following properties to `BTVenmoRequest`
+    * `collectCustomerBillingAddress`
+    * `collectCustomerShippingAddress`
+    * `totalAmount`
+    * `subTotalAmount`
+    * `discountAmount`
+    * `taxAmount`
+    * `shippingAmount`
+    * `lineItems`
+  * Add new `isVenmoEnrichedCustomerDataEnabled` field to `BTConfiguration+Venmo`
```

Comment:
```suggestion
```

We can remove this since it's internal

Reply:
Uhhh I was pretty sure I removed it but all this rebasing and trying to maintain commit history is chaotic 😅